### PR TITLE
#165276369 fix clap count

### PR DESCRIPTION
**Pull request by Chiazokam**
#### Description
- Fixed the getArticleClaps method so as to return a count

#### Type of change
- [x] Bug(none breaking change)
- [x] write tests for the getArticleClaps method
- [x] modify sequelize query

#### PIVOTAL BOARD
[#165276369](https://www.pivotaltracker.com/n/projects/2320805/stories/165276369)

#### SCREENSHOT
None

### CODE REVIEW

File: server/utils/rating.util.js [hunk 1]
```diff
@@ -9,19 +9,19 @@ class Rating {
    * @returns {number} claps
    */
   static async getArticleClaps(articleId) {
-    const claps = await Clap.findAll({
+    const claps = await Clap.findAndCountAll({
       where: { articleId }
     });
-    return claps;
+    return claps.count;
```
Comment (paradisekelechi) [inline server/utils/rating.util.js]: This method is `getArticleClaps` and not `getArticleClapsCount`.  Hence, it should return the claps and not the count. @Chiazokam

File: server/utils/rating.util.js [hunk 2]
```diff
@@ -9,19 +9,19 @@ class Rating {
    * @returns {number} claps
```
Comment (paradisekelechi) [inline server/utils/rating.util.js]: It returns an object and not a number

**Review by paradisekelechi — CHANGES_REQUESTED** [review]

File: server/utils/rating.util.js (hunk 1 above)
Comment (Chiazokam) [reply server/utils/rating.util.js]: Hi @paradisekelechi  Please I am not really clear on the feedback. Should I rename the method?
Or should I remove the count property and put it where I am calling the method?

Comment (paradisekelechi) [reply server/utils/rating.util.js]: @Chiazokam Would it make sense for you to use `findAndCountAll` and then return just the `count`? Wouldn't that result in limiting ourselves and the data?

Comment (Chiazokam) [reply server/utils/rating.util.js]: Alright...I understand. Thanks

File: server/utils/rating.util.js (hunk 2 above)
Comment (Chiazokam) [reply server/utils/rating.util.js]: I've fixed this

**Review by paradisekelechi — APPROVED** [review]

**Pull request merged**